### DISCUSSION TITLE
4.3.52

### DISCUSSION THREAD

**Discussion by KillerInk** [KillerInk/FreeDcam#176] [General]
-bug fixes #175 #166 #169
-add custom ae with ability to use iso/shutter priority when using glpreview.

<hr /><em>This discussion was created from the release <a href='https://github.com/KillerInk/FreeDcam/releases/tag/'>4.3.52</a>.</em>

**Comment by fob014** [comment]
even after scene look completely focused in view finder 
it still take almost  1 to 1.5   sec delay to capture photo ???
camera 2api
android 11

**Comment by Wayneaswell** [comment]
Found this link hidden down bottom of page, but still will not install.  How do we install please?

Thank you.
